### ISSUE TITLE
Usage on alpine docker image

### ISSUE THREAD

**Issue by spdowling** [hashicorp/terraform-provider-helm#59]
Currently it appears that the provider is compiled with required support for glibc, which Alpine doesn't have prepared OOTB in their main image.

Checking in to the linked libraries shows up as:

```
/go/terraform-provider-helm_linux_amd64 # ldd terraform-provider-helm
	/lib64/ld-linux-x86-64.so.2 (0x7f3e5a379000)
	libpthread.so.0 => /lib64/ld-linux-x86-64.so.2 (0x7f3e5a379000)
	libc.so.6 => /lib64/ld-linux-x86-64.so.2 (0x7f3e5a379000)
```

Could we get a compiled release version that would allow us to run the provider inside an Alpine-derived docker container or other containers without glibc?

**Comment by spdowling** [comment]
I did manage to get it running for now using an Alpine image with some tweaking to get glibc support built-in.

https://hub.docker.com/r/frolvlad/alpine-glibc/
https://github.com/frol/docker-alpine-glibc

```
FROM frolvlad/alpine-glibc:alpine-3.7
```

**Comment by stevenaldinger** [comment]
thank you so much @spdowling, I run it inside of an alpine container too. saved me a heartbreak

**Comment by subesokun** [comment]
Any updates on this issue? I've the same problem but I'm in the situation that I can't change the base image.

**Comment by stevenaldinger** [comment]
you should be able to apk install `libc6-compat` but I haven't tested

**Comment by obeyler** [comment]
tested and it works fine
see 
https://github.com/orange-cloudfoundry/paas-docker-cloudfoundry-tools/blob/master/terraform/Dockerfile

**Comment by navinkbe7** [comment]
@stevenaldinger Adding libc6-compat worked for me . 
Additional Info : I had tested it on `python:2.7-alpine3.6`

**Comment by meyskens** [comment]
Should be fixed by #111
